layers/meta-resin-raspberrypi/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,18 @@
+RESIN_EXTERNAL_MMC = "1"
+RESIN_INTERNAL_MMC = "0"
+UBOOT_KCONFIG_SUPPORT = "1"
+inherit resin-u-boot
+
+DEPENDS += "coreutils-native"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+
+SRC_URI += "file://uEnv.txt_internal \
```

Comment:
As far as I remember this was only needed for flasher types. It shouldn't be needed for any raspberrypi ones.

Reply:
I see. will remove it